src/zip.js

```diff
@@ -22,28 +26,43 @@ const zipFunctions = async function (
     jsExternalModules = [],
     jsIgnoredModules = [],
     parallelLimit = DEFAULT_PARALLEL_LIMIT,
-    skipGo,
+    skipGo = true,
     zipGo,
   } = {},
 ) {
-  const [srcPaths, pluginsModulesPath] = await Promise.all([getSrcPaths(srcFolder), getPluginsModulesPath(srcFolder)])
+  await makeDir(destFolder)
```

Comment:
Should this be run in parallel to `listFunctionsDirectory()`? For performance.

Reply:
Definitely! Done in https://github.com/netlify/zip-it-and-ship-it/pull/364/commits/c0fe24f5650b41742b2fae70d1a4c02eb8b4eb82.